book/memory/TEA.md

```diff
@@ -0,0 +1,128 @@
+## Level 2.5 - TEA TIME
+
+The Elm architecture is made up of three parts: `model`, `view` and `update`.
+
+##### Model
+The model is the state of our application. In Elm this is usually modeled with a record, defined by using a `type alias`.
+
+##### View
+A function that takes the current application state as an argument and returns some HTML.
+
+##### Update
+A function that takes a "message", the current application state and returns an updated application state.
+This message is modeled with a union type, so usually the `update` function consists of pattern matching on the message value. 
```

Comment:
Denne likte jeg, konkret og fin  "so usually the `update` function consists of pattern matching on the message value. "